host/3.0/buster/amd64/python/python37/python37.Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN EXTENSION_BUNDLE_VERSION=1.8.1 && \
     rm -f /$EXTENSION_BUNDLE_FILENAME_V3 &&\
     find /FuncExtensionBundles/ -type f -exec chmod 644 {} \;
 
-FROM python:3.7-slim-buster
+FROM mcr.microsoft.com/mirror/docker/library/python:3.7-buster
```

Comment:
Same thing here. Likely the same for all images. Won't callout others. 
Previously this relied on Slim. Now it relies on fat

Reply:
Changed my mind. Please change this to -slim to minimize changes. Will keep parity between this and appservice images. Please do this for all non-suffix images. Ideally in a single commit